lib/jolie-js/src/jolie/js/JsUtils.java

```diff
@@ -224,4 +272,32 @@ public static void parseJsonIntoValue( Reader reader, Value value, boolean stric
 			throw new IOException( e );
 		}
 	}
+	
+	public static void parseNdJsonIntoValue( Reader reader, Value value, boolean strictEncoding )
+		throws IOException
+	{
+		String stringBuffer = new BufferedReader( (InputStreamReader) reader ).lines().collect( Collectors.joining( "\n" ) );
```

Comment:
I don't like this explicit cast to `InputStreamReader`. If you need a BufferedReader, get a BufferedReader already in the type of the method.